File: coursera/algorithmic-toolbox/week-1/programming-assignment-1-sum-of-two-digits/solution.go
package main

import (
	"bufio"
	"fmt"
	"io"
	"os"
	"strconv"
)

func main() { sumOfTwoDigits(os.Stdin, os.Stdout) }

func sumOfTwoDigits(reader io.Reader, writer io.Writer) {
	scanner := bufio.NewScanner(reader)
	scanner.Split(bufio.ScanWords)

	_ = scanner.Scan()
	a, _ := strconv.Atoi(scanner.Text())
	_ = scanner.Scan()
	b, _ := strconv.Atoi(scanner.Text())

	safe(fmt.Fprintln(writer, a+b))
}

func safe(interface{}, error) {}

File: coursera/algorithmic-toolbox/week-1/programming-assignment-1-sum-of-two-digits/solution.py
# python3
import sys


def sum_of_two_digits(reader, writer):
    a, b = map(int, reader.readline().split())
    writer.write(str(a + b) + '\n')


if __name__ == '__main__':
    sum_of_two_digits(sys.stdin, sys.stdout)
